src/slurm_plugin/clustermgtd.py

```diff
@@ -717,6 +729,7 @@ def _find_unhealthy_slurm_nodes(self, slurm_nodes):
                     ).append(node)
                 else:
                     unhealthy_dynamic_nodes.append(node)
+        self._event_publisher.publish_unhealthy_node_events(all_unhealthy_nodes)
```

Comment:
currently, we are not publishing events when there's no unhealthy nodes since we did not set  kwargs in both clustermgtd and slurm_resume. for emit_event(event_level, message, event_type, timestamp=None, event_supplier=None, **kwargs).
Will it be better to add a check for `if all_unhealthy_nodes` so we can skip going through `invalid_backing_instance_nodes` and `publish_event` to make it more efficient?

Reply:
I updated the code in publish_unhealthy_node_events so that it will basically noop if all_unheathy_nodes is empty if the log level is higher than debug.